tests/autorestart/test_container_autorestart.py

```diff
@@ -26,7 +26,7 @@
 DHCP_RELAY = "dhcp_relay"
 DHCP_SERVER = "dhcp_server"
 POST_CHECK_INTERVAL_SECS = 1
-POST_CHECK_THRESHOLD_SECS = 360
+POST_CHECK_THRESHOLD_SECS = 600
```

Comment:
should use different threshold for T0/T1?

Reply:
i have added new variable so that it wont affect the worst case for t0/t1